.github/workflows/conformance-ipsec-e2e.yaml

```diff
@@ -199,6 +201,19 @@ jobs:
           kind-params: "${{ steps.kind-params.outputs.params }}"
           kind-image: ${{ env.KIND_K8S_IMAGE }}
 
+      - name: Start Cilium KVStore
+        id: kvstore
+        if: matrix.kvstore == 'true'
```

Comment:
do we need to do ${{ }}

Reply:
It is not strictly necessary for simple expressions: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/using-conditions-to-control-job-execution#overview

Reply:
ah nice TIL, thanks.